README.md

```diff
@@ -17,20 +17,21 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
-[] Undefined
+[X] Strings
+[X] Booleans
+[X] Undefined
 [] NaN
-[] Integers
-[] Arrays
-[] Null
+[X] Integers
+[X] Arrays
+[X] Null
 ```
```

Comment:
:+1: A couple of notes (none of which I'm docking you for!)...

* Arrays are technically a data collection -- not a data type. That being said, you use them so much they might as well be!
* An integer is technically not a data type, but a "Number" is.